contrib/IECoreUSD/test/IECoreUSD/SceneCacheFileFormatTest.py

```diff
@@ -295,7 +295,7 @@ def testTagsLoadedAsCollections( self ):
 		tags = { "t1" : [ "/t" ], "s1" : [ "/t/s" ], "all" : [ "/t", "/t/s" ] }
 		for tag, paths in tags.items():
 			relationship = tagPrim.GetRelationship( "collection:{}:includes".format( tag ) )
-			sdfPaths = map( lambda x: pxr.Sdf.Path( x ), paths )
+			sdfPaths = [ pxr.Sdf.Path( x ) for x in paths ]
```

Comment:
just to confirm, this is an alternative to casting the map ( which I presume returns a generator in python 3 ) as a list?